bundles/action/org.openhab.action.ciscospark/README.md

```diff
@@ -0,0 +1,43 @@
+# Cisco Spark Actions
+
+[Cisco Spark](https://www.ciscospark.com) can be used to send messages to groups or individuals 
+when certain events take place in openHAB. 
+
+Messages in the Cisco Spark plugin support Markdown text.
+
+
+## Actions
+
+* `sparkMessage('message', 'roomId')`: Send a message to a specific room.
+* `sparkMessage('message')`: Send a message to the default room (from config)
+* `sparkPerson('message', 'personEmail)`: Send a direct message to a person
+
+## Configuration
+
+The Cisco Spark action service is one of the pre-defined set of actions that is automatically 
+available, and can be used within rules and scripts. Before you can use Cisco Spark you need 
+to register your application with the Cisco Spark Developers service.
+
+### Register your application
+
+Login and navigate to Cisco [Spark for Developers](https://developer.ciscospark.com/apps.html).  
+You are presented with two options: create an integration or a Bot.  Select 'Create a Bot'.
+
+Follow the screens to create your bot.  Once done you'll be presented with an access token.
+
+### Configure openhab
+
+Configure openhab with the access token by adding the following to the file `services/ciscospark.cfg`:
```

Comment:
@9037568 : The README.md is not concerned with OH1 runtime users, and should reference the .cfg file used in openHAB 2.  If there is an expectation that this action will be used by OH1 runtime users (and why not), the body of README.md can be copied to a wiki page and adjustments can be made for OH1 users.  Or, OH1 runtime users can just figure it out.